launch_xml/test/launch_xml/test_arg.py

```diff
@@ -34,3 +36,18 @@ def test_arg():
     assert 'my_arg' == arg.name
     assert 'asd' == ''.join([x.perform(None) for x in arg.default_value])
     assert 'something' == arg.description
+
+
+def test_arg_wrong_attribute():
+    xml_file = \
+        """\
+        <launch>
+            <arg name="my_arg" whats_this="hello" default="asd" description="something"/>
+        </launch>
+        """
+    xml_file = textwrap.dedent(xml_file)
+    root_entity, parser = Parser.load(io.StringIO(xml_file))
+    with pytest.raises(ValueError) as excinfo:
+        parser.parse_description(root_entity)
+    assert '`arg`' in str(excinfo.value)
+    assert 'whats_this' in str(excinfo.value)
```

Comment:
@ivanpauno can we add a test for a more complex (i.e. nested) description?

Reply:
See https://github.com/ros2/launch/pull/468/commits/9dfb1a7a7cf575b4193c32b9753fbc958cdaddb9.